tests/pytests/integration/modules/test_mac_keychain.py

```diff
@@ -0,0 +1,134 @@
+"""
+Validate the mac-keychain module
+"""
+
+import os
+
+import pytest
+
+import salt.utils.versions
+from salt.exceptions import CommandExecutionError
+from tests.support.runtests import RUNTIME_VARS
+
+pytestmark = [
+    pytest.mark.slow_test,
+    pytest.mark.destructive_test,
+    pytest.mark.skip_if_not_root,
+    pytest.mark.skip_unless_on_darwin,
+]
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_vars(salt_call_cli, base_env_state_tree_root_dir):
```

Comment:
```suggestion
def setup_teardown_vars(salt_call_cli, base_env_state_tree_root_dir):
```
I would have named this fixture differently, like, `cert_details` or something alone these lines, would probably also create a different object:
```python
cert_details = types.SimpleNamespace(cert=<the path>, alias=<the alias>, passwd=<the passwd>)
return cert_details
```
But, not a blocker.